gpsd_client:
  ros__parameters:
    use_gps_time: false
    check_fix_by_variance: false
    frame_id: gps
    publish_rate: 10
